nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
   - mkdir ${TRAVIS_BUILD_DIR}/tests && cd ${TRAVIS_BUILD_DIR}/tests
 
 env:
-  - NXF_VER='0.31.1' # Specify a minimum NF version that should be tested and work
+  - NXF_VER='0.32.0' # Specify a minimum NF version that should be tested and work
```

Comment:
For next time 😉 

```
nf-core bump-version -n . 0.32.0
```

Reply:
I have changed the relevant files back to 0.31.1 to do it as you suggested. Does the 'nf-core' command come from an 'nf-core/tools' install? "

Error: No such command "bump-version"."

Reply:
Is it "nf-core release"?

Reply:
> I have changed the relevant files back to 0.31.1 to do it as you suggested. 

Hmm? Where did I suggest that? I think we need `0.32.0` if we're now using the new manifest stuff in the template.

> Does the `nf-core` command come from an `nf-core/tools` install?

Yup! See http://nf-co.re/tools or the readme

> `Error: No such command "bump-version".`

Really? Ah could be because it's new since the last release possibly? How did you install it? Make sure you get the development version from GitHub and install it locally - see the above docs link.

The `nf-core bump-version` tool simply updates the requested version number (pipeline or required nextflow) in all of the different places needed. I wrote it because it's a hassle to change manually without forgetting somewhere..

```
$ nf-core bump-version --help

                                          ,--./,-.
          ___     __   __   __   ___     /,-._.--~\
    |\ | |__  __ /  ` /  \ |__) |__         }  {
    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                          `._,._,'

Usage: nf-core bump-version [OPTIONS] <pipeline directory> <new version>

  Update nf-core pipeline version number

Options:
  -n, --nextflow  Bump required nextflow version instead of pipeline version
  --help          Show this message and exit.
```

Reply:
> Is it "nf-core release"?

Ah - it may have been called that in a earlier version of the code, yes. But you'll need the dev version to do required nextflow version numbers anyway.

```
pip install --upgrade --force-reinstall git+https://github.com/nf-core/tools.git
```

Reply:
Ok. Cool. Yes, I installed the latest release https://github.com/nf-core/tools/archive/1.1.tar.gz. We use environment modules which makes it a pain to install things via pip. Had to install it manually. Ill have to do the same with the dev version. We are working on getting a local install of Python with the correct permissions to be able to use pip. Ill give it a go.

Reply:
Aha - just use miniconda, that's what we do. You can install it under your own user and then you have Python without the need for `sudo` (and so you can easily use `pip`). See [miniconda downloads](https://conda.io/miniconda.html). Then just make an environment for yourself and always activate it with your `.bashrc` file (mine is helpfully called `work`).

Reply:
Ah sorry, it's on the `dev` branch: https://github.com/nf-core/tools/blob/2d22c8d2afca6532f42c9c99302168d9c0d4138b/scripts/nf-core#L117-L146

```bash
git clone https://github.com/nf-core/tools.git
cd tools
git checkout dev       # <-- new line
pip install --upgrade --force-reinstall .
```

Reply:
Shouldnt there be an "import sys" statement in this file? I have put it in for now in my local install.

https://github.com/nf-core/tools/blob/dev/nf_core/bump_version.py

Reply:
Probably, yes!